pkg/config/config.go

```diff
@@ -1089,6 +1076,27 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 	return nil
 }
 
+// ValidateDefault ensures that the default runtime is set and valid.
+func (c *RuntimeConfig) ValidateDefault() error {
+	// check we do have at least a runtime
+	if _, ok := c.Runtimes[c.DefaultRuntime]; !ok {
+		// Set the default runtime to "runc" if default_runtime is not set
+		if c.DefaultRuntime == "" {
```

Comment:
nit, can we invert this condition and compress the else?

`if c.DefaultRuntime != "" { return } ...`

Reply:
I suppose this was already there, but doesn't hurt to update now 🙂